classes/class4/hw/public/javascripts/kitchen.js

```diff
@@ -0,0 +1,17 @@
+$("[type='button']").click(function(event){
```

Comment:
I would give these buttons a class to target rather than finding any element with `type='button'`